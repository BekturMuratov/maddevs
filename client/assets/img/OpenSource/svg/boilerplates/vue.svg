<svg xmlns="http://www.w3.org/2000/svg" width="336" height="125" fill="none" xmlns:v="https://vecta.io/nano"><g clip-path="url(#A)"><path d="M206.784.001l-38.797 67.203L129.191.001H-.008l167.996 290.997L335.984.001h-129.2z" fill="#41b883"/><path d="M206.785.001l-38.797 67.203L129.191.001h-62l100.798 174.595L268.786.001h-62.001z" fill="#34495e"/></g><defs><clipPath id="A"><path fill="#fff" d="M0 0h336v291H0z"/></clipPath></defs></svg>