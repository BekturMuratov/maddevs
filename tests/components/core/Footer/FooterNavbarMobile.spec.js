import FooterNavbarMobile from '@/components/core/Footer/FooterNavbarMobile'
import { render } from '@testing-library/vue'

describe('FooterNavbarMobile component', () => {
  it('should render correctly', () => {
    const { container } = render(FooterNavbarMobile)

    expect(container).toMatchSnapshot()
  })

  it('should render with current year', () => {
    const { html } = render(FooterNavbarMobile)

    expect(html()).toContain(` Â© Mad Devs - ${new Date().getFullYear()}`)
  })
})
