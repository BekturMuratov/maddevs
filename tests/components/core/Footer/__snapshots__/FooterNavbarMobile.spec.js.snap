// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterNavbarMobile component should render correctly 1`] = `
<div>
  <div
    class="footer-navbar-mobile"
  >
    <p
      class="footer-navbar-mobile__company-name"
    >
      
    Â© Mad Devs - 2021
  
    </p>
     
    <div
      class="footer-navbar-mobile__nav-list"
    >
      <a
        class="footer-navbar-mobile__nav-link"
        href="/gdpr/"
        target="_blank"
      >
        
      GDPR Compliance Commitment
    
      </a>
      <a
        class="footer-navbar-mobile__nav-link"
        href="/nda/"
        target="_blank"
      >
        
      Non-Disclosure Agreement (NDA)
    
      </a>
      <a
        class="footer-navbar-mobile__nav-link"
        href="/privacy/"
        target="_blank"
      >
        
      Privacy Policy
    
      </a>
      <a
        class="footer-navbar-mobile__nav-link"
        href="/faq/"
        target="_blank"
      >
        
      FAQ
    
      </a>
    </div>
  </div>
</div>
`;
